test/e2e_env/multizone/sync/sync.go

```diff
@@ -217,5 +217,24 @@ spec:
 				Expect(err).To(HaveOccurred())
 			})
 		})
+
+		It("should sync policy with a long name and store it as display name", func() {
```

Comment:
Doesn't it make sense to add one with a pod with the longest possible name to catch future bugs like this.

Reply:
So adding Pod with 253 chars ends up with error on my mac
```
E0212 12:45:49.211145       7 pod_workers.go:965] "Error syncing pod, skipping" err="failed to \"CreatePodSandbox\" for \"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx_sync(4397c70d-6737-4045-964b-ff9ef9f9d41a)\" with CreatePodSandboxError: \"Failed to create log directory for pod \\\"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx_sync(4397c70d-6737-4045-964b-ff9ef9f9d41a)\\\": mkdir /var/log/pods/sync_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx_4397c70d-6737-4045-964b-ff9ef9f9d41a: file name too long\"" pod="sync/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" podUID=4397c70d-6737-4045-964b-ff9ef9f9d41a
```

Seems like hit limit of file name length HFS and extFS has a limit of 255. Not sure if this is K3D thing or Kubernetes overall (creating this filename). Do you still want me to add a test with ~200 chars?

Reply:
Lol :) I'm curious if this sort of test passes on Linux. It's likely ok to ignore this test for now to not block progress on this here.